cmd/crank/configuration/configuration.go

```diff
@@ -96,12 +103,35 @@ type CreateCmd struct {
 	Name    string `arg:"" name:"name" help:"Name of Configuration."`
 	Package string `arg:"" name:"package" help:"Image containing Configuration package."`
 
-	History  int  `help:"Revision history limit."`
-	Activate bool `short:"a" help:"Enable automatic revision activation policy."`
+	RevisionHistoryLimit int64 `help:"Revision history limit."`
+	ManualActivation     bool  `short:"a" help:"Enable manual revision activation policy."`
```

Comment:
I switched this because it seems the default is automatic in the reconciler while if user skips giving `Activate` a value, it'd actually mean `false` hence manual activation will be chosen, which is a different behavior than YAML apply.